### PR TITLE
Adding optional Monetization demo add-on

### PR DESCRIPTION
**Pull request by danistrebel**
Suggesting:
* Mint Policy that kicks in when the currency proxy is used in a product that has is part of a rateplan.
* Script to create product/developer/app that use mint in POSTPAID

### CODE REVIEW

File: deploy-mint.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#echo "Enabling APIs..."
+#gcloud services enable cloudbuild.googleapis.com
+
```
Comment (ssvaidyanathan) [inline deploy-mint.sh:19]: @danistrebel - few questions

1. How is this script called? Will `deploy-apigee.sh` call this?
2. If this is called separately, we will need to download the apigeecli as the other scripts deletes the apigeecli once complete
3. We will also need a delete-mint.sh to cleanup

Comment (danistrebel) [reply deploy-mint.sh:19]: Good input. I please see the new commit.

File: README.md [hunk 2]
```diff
@@ -129,6 +137,7 @@ For each fo the "deploy" scripts, there is an equivalent "cleanup" script you ca
 If you want to clean the entire setup, first source your `env.sh` script, and then run the scripts in the following order
 
 ```bash
+./cleanup-mint.sh # if applicable
```
Comment (ssvaidyanathan) [inline README.md:154]: Can we include the same in the docs/cloudshell-tutorial.md as well?

Comment (danistrebel) [reply README.md:154]: good shout. Done.

**Pull request merged**